### ISSUE TITLE
Error sending data to Zooniverse

### ISSUE THREAD

**Issue by jsv1206** [lsst-epo/citizen-science-notebooks#29]
**Describe the bug**

I get the error when I tried to send image cutouts to zooniverse with send_data(subject_set_name, batch_dir, cutouts)

However, I can successfully send data from Citizen_Science_testing.ipynb notebook.

I have the metadata.csv in the cutout folder in the same format as Citizen_Science_testing.ipynb

**To Reproduce**
Steps to reproduce the behavior, written in imperative mood:
1. Go to 'variable_stars' branch
2. Click on 'variable_stars_test.ipynb'
3. Scroll down to 'Send data to Zooniverse' cell in the end of the notebook

**Expected behavior**
Expect to run the cell without error 

**Actual behavior**
Get an error as shown below in the sample output

**EDC Output**

```
SAMPLE OUTPUT
'1. Checking batch status'
'2. Writing metadata file required by the Rubin EPO Data Center.'
'3. Zipping up all the astro cutouts - this can take a few minutes with large data sets, but unlikely more than 10 minutes.'
'4. Uploading the citizen science data'
'5. Creating a new Zooniverse subject set'
'6. Notifying the Rubin EPO Data Center of the new data, which will finish processing of the data and notify Zooniverse'
'7. Cleaning up unused subject set on the Zooniverse platform, vendor_batch_id : 112178'
'8. Cleaning up unused subject set on the Zooniverse platform, vendor_batch_id : 112178'
"** Warning: Failed to find the subject set with id: 112178- perhaps it's been deleted?."
root ERROR: ** One or more errors occurred during the last step **
root ERROR: An error occurred while processing the data transfer process upload
root ERROR: Email address: jsv1206@gmail.com
root ERROR: Timestamp: 2023-03-15 12:26:45.552636-07:00
```

**Additional context**

**Comment by ericdrosas87** [comment]
This should be fixed with #30

**Comment by jsv1206** [comment]
I can now send individual images (not flipbook) from variable_stars.ipynb
